index.js

```diff
@@ -27,16 +40,17 @@ exports.createApiAction = (apiBaseUrl, method, endpoint) => {
 
         const headers = {
             'Accept': 'application/json',
-            'Content-Type': 'application/json',
             'Authorization': (getState().auth && getState().auth.token) ? "Bearer " + getState().auth.token : ''
         }
 
+        const body = options.files ? getFormData(options) : JSON.stringify(options.body)
```

Comment:
JSON.stringify(options.body)

is not equivalent to 

options.body ? JSON.stringify(options.body) : ''

Reply:
Oops, thought that it was, my bad.